### ISSUE TITLE
Please release a new version

### ISSUE THREAD

**Issue by jmairboeck** [AmpScm/AnkhSVN#57]
The latest available build from Github Actions for VS 2022 has expired. The current released version lacks a few important bug fixes. There is currently no recent build available for VS 2022.

**Comment by giaphungvn** [comment]
good job! We need it, thanks!

**Comment by agausachs** [comment]
Yes please. At least to fix the incompatibilty with Visual Studio 2022 >=17.2.
Thanks!!

**Comment by rhuijben** [comment]
New build created under actions. Will try to release this version via VS portal.

**Comment by agausachs** [comment]
> New build created under actions. Will try to release this version via VS portal.

Thank you Bert! Now I can see the new artifact under Actions, but only for 2017-2019. It seems the build failed building for vs2022.

_"C:\Users\runneradmin\.nuget\packages\microsoft.vssdk.buildtools\17.0.5232\tools\VSSDK\Microsoft.VsSDK.targets(829,5): error MSB4018:  [D:\a\AnkhSVN\AnkhSVN\build\src\Ankh.Package\Ankh.Package.csproj]
Error: Process completed with exit code 1."_

**Comment by Celess22** [comment]
Looks like build 2019 failed, just after the 2017-2019 one, and never got to the 2022 one.  I really appreciate all your work on this.

Maybe is some way to just run 2022. Or maybe it's the dependency downloads at all.

**Comment by Seph88666** [comment]
any news on the build for VS2022? Thanks a lot!

**Comment by miba222** [comment]
Oh yes, please create a new vsix for VS2022. Microsoft urged me to switch from 2019 to 2022, and now I am heavily missing my AnkhSVN plugin...

**Comment by ARMoir** [comment]
I built the attached from source on my VS2022 install and was able to install the VSIX once built. Hope this works for you and can hold people over until there is a new release!
[Release2022.zip](https://github.com/AmpScm/AnkhSVN/files/10095470/Release2022.zip)

**Comment by miba222** [comment]
@ARMoir First impression looks good. Thank you so much!!!

**Comment by ARMoir** [comment]
I forked the repo [https://github.com/ARMoir/AnkhSVN](https://github.com/ARMoir/AnkhSVN) and I am going to be making some small updates and hopfully my pull request gets approved.

**Comment by Blackpaw** [comment]
> I forked the repo https://github.com/ARMoir/AnkhSVN and I am going to be making some small updates, hopefully my pull request gets approved.

Thanks - will this install happily over the official 2019 build?

**Comment by ARMoir** [comment]
> Thanks - will this install happily over the official 2019 build?

I believe it should, but please let me know what you find.

**Comment by Blackpaw** [comment]
That installed ok and seems to be operating fine, but I do have two entries for AnkhSVN in extension manager now. The old one is ver 2.9.87, the new install is 2.9.3

I uninstalled the old version (2.9.87) and all seems ok.

**Comment by Blackpaw** [comment]
> I believe it should if you use the Ankh-VSIX-2017-2019 Artifact

Update - I misunderstood your directions, I was using 2019 in VS2022 and installed your 2022 over the top of it, I guess that's why I had two versions in Extension manager. Regardless, it works fine, thanks.

**Comment by ARMoir** [comment]
@Blackpaw No problem! Yeah the build numbers reset on the github actions when I forked the repo, so that's why it went from 87 to 3. I made a few more updates and I am only building for 2022 now, as that is what I am using, glad if it helps!

**Comment by nikeruga** [comment]
Where can I get the new version?

**Comment by ARMoir** [comment]
@nikeruga 
I ran a build 2/5/23 on my branch
https://github.com/ARMoir/AnkhSVN/actions/runs/4099889922

**Comment by vman2000** [comment]
> I forked the repo https://github.com/ARMoir/AnkhSVN and I am going to be making some small updates, hopefully my pull request gets approved.

Thank you, early days but your fork seems to resolve the instability issues I was getting on main.

**Comment by ARMoir** [comment]
@vman2000 I have found that when I add the latest updates to the branch it does then require Visual Studio to be updated to the latest version, what instability issues are you encountering?

**Comment by BearSleepy** [comment]
Here's a 2022 build using more direct means.

It's from my AddOn Studio releases basically, which now having to build AnkhSVN also. The reason couldn't include the other one is because it effectively sets the minimum VS version at 17.5.

This is built in a more basic way, that's more in-line with the setup for other VS components.

https://github.com/BearSleepy/AnkhSVN/releases

**Comment by BearSleepy** [comment]
SS from actual Visual Studio. 
![image](https://github.com/AmpScm/AnkhSVN/assets/17802598/63a3e446-9540-417b-ae91-a450c2c10118)

Seems to work.

**Comment by BearSleepy** [comment]
![image](https://github.com/AmpScm/AnkhSVN/assets/17802598/55393a44-742b-4e88-bdee-9e16354709f1)

**Comment by BearSleepy** [comment]
Updated link in case the github redirect goes away. Moved to the same umbrella as AddOn Studio.

https://github.com/FallenWorlds/AnkhSVN

**Comment by ARMoir** [comment]
https://github.com/AmpScm/AnkhSVN/releases#release-v2.9.191
